spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/BoxControllerHelperService.java

```diff
@@ -676,6 +682,61 @@ public JSONObject prepareBoxScanner(HttpSession session, JSONObject json) {
     }
   }
 
+  private static boolean isRealPosition(BoxScan scan, String position) {
+    return !position.equals(scan.getNoReadLabel()) && !position.equals(scan.getNoReadLabel());
```

Comment:
Checks same condition twice. Is one supposed to be `getNoTubeLabel`? Either of these conditions is still a real position, so maybe "isFilledPosition" or something would be a better method name